### PR TITLE
Fix: Review comments

### PR DESCRIPTION
**Pull request by morten-holm**

### CODE REVIEW

File: src/main/java/com/kapeta/spring/config/KapetaSecurityProviderConfig.java [hunk 1]
```diff
@@ -26,11 +38,44 @@ public SecurityFilterChain kapetaFilterChain(HttpSecurity http) throws Exception
...
                         .anyRequest().authenticated()
                 );
+
+        return http.build();
+    }
+
+    @Bean
+    public SecurityFilterChain jwksFilterChain(HttpSecurity http) throws Exception {
+        http
+                .securityMatcher(KapetaJwksRestController.PATH_WELL_KNOWN_JWKS)
```
Comment (morten-holm) [inline src/main/java/com/kapeta/spring/config/KapetaSecurityProviderConfig.java:46]: This ensures that this particular security chain only applies to this specific path

File: src/main/java/com/kapeta/spring/config/KapetaSecurityProviderConfig.java [hunk 2]
```diff
@@ -24,13 +35,44 @@ public SecurityFilterChain kapetaFilterChain(HttpSecurity http) throws Exception
...
+    @Bean
+    @ConditionalOnMissingBean(JWKInternalKeyStoreProvider.class)
+    public JWKInternalKeyStoreProvider jwksPublicKeyProvider() {
+        log.info("Creating JWKS Public Key");
+        try {
+            RSAKey jwk = new RSAKeyGenerator(2048)
+                    .keyUse(KeyUse.SIGNATURE)
+                    .keyID(UUID.randomUUID().toString())
+                    .generate();
+            return () -> new JWKInternalKeyStore("https://auth.kapeta", "https://auth.kapeta", new JWKSet(jwk));
```
Comment (hofmeister) [inline src/main/java/com/kapeta/spring/config/KapetaSecurityProviderConfig.java]: Let's go with 
```
issuer: 'https://example.auth.kapeta.com',
audience: 'https://example.kapeta.com',
```

Fixing that in nodejs as well

**Review by hofmeister — CHANGES_REQUESTED** [review]
Otherwise 👍

File: src/main/java/com/kapeta/spring/config/KapetaSecurityProviderConfig.java (hunk 2 above)
Comment (hofmeister) [reply src/main/java/com/kapeta/spring/config/KapetaSecurityProviderConfig.java]: https://github.com/kapetacom/codegen-target-nodejs/pull/54

Comment (morten-holm) [reply src/main/java/com/kapeta/spring/config/KapetaSecurityProviderConfig.java]: Done

**Pull request merged**